<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="test_eos_exception_bytes.TestEosExceptionBytes-20200216171907" tests="1" file="test_eos_exception_bytes.py" time="0.000" timestamp="2020-02-16T17:19:07" failures="0" errors="0" skipped="0">
	<testcase classname="test_eos_exception_bytes.TestEosExceptionBytes" name="test_eos_exception_bytes" time="0.000" timestamp="2020-02-16T17:19:07" file="spec/python/test_eos_exception_bytes.py" line="9">
		<system-out><![CDATA[]]></system-out>
		<system-err><![CDATA[]]></system-err>
	</testcase>
</testsuite>
